srcpkgs/drumstick/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'drumstick'
+pkgname=drumstick
+version=1.1.2
+revision=1
+build_style=cmake
+configure_args="-DLIB_SUFFIX="
+hostmakedepends="pkg-config doxygen libxslt docbook-xsl"
+makedepends="qt5-svg-devel alsa-lib-devel pulseaudio-devel"
+short_desc="a set of MIDI libraries using C++/Qt5 idioms and style"
+maintainer="Mohammad Amin Sameti <mamins1376@gmail.com>"
+license="GPL-3.0-or-later"
```

Comment:
This is a lie. Please get this right.
`GPL-2.0-or-later`

Reply:
From the homepage:

`License: GPL v2 or later`